### PR TITLE
[ADD] New module pos_session_report

### PR DESCRIPTION
**Pull request by hendrixcosta**
This module extends the functionality of Pos Session Sumary and to allow you to show detailed informations about pos session

### CODE REVIEW

**Comment by mileo** [comment]
- Please fix blank spaces in the end of files;
- Some portuguese strings
- Travis errors

**Comment by pedrobaeza** [comment]
@legalsylvain, isn't `pos_session_summary` the same functionality?

**Comment by rvalyi** [comment]
pointer https://github.com/OCA/pos/pull/77

File: pos_session_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Pos Session Report Detailed
+==============
+
+This module extends the functionality of Pos Session Sumary
+and to allow you to show detailed informations about pos session
```
Comment (rafaelbn) [inline pos_session_report/README.rst]: Which information?

File: pos_session_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
```
Comment (rafaelbn) [inline pos_session_report/README.rst]: If there is not special considerations to take in care just write it "normal instalation"

File: pos_session_report/README.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+#. Go to ...
+
+.. figure:: path/to/local/image.png
+   :alt: alternative description
+   :width: 600 px
+
+Usage
+=====
+
+To use this module, you need to:
```
Comment (rafaelbn) [inline pos_session_report/README.rst]: Please explain how to use / test module. What is expected, which info?

**Comment by rafaelbn** [comment]
Hi @Hendrixcosta , thanks for this contribution. Please check module https://github.com/OCA/pos/tree/9.0/pos_session_summary to see if it's what you need.

> "This module extends the functionality of Pos Session Sumary and to allow you to show detailed informations about pos session"

Could you paste a screenshot like https://github.com/OCA/pos/pull/77#issue-130150019

**Review by rafaelbn — COMMENTED** [review]
Still pending comments from @mileo https://github.com/OCA/pos/pull/122#issuecomment-251808282

File: pos_session_report/README.rst (hunk 1 above)
Comment (legalsylvain) [reply pos_session_report/README.rst]: @Hendrixcosta, indeed, a print screen of a part of the generated pdf could be great to explain clearly what this module does.

File: pos_session_report/i18n/pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: pos_journal_report
+#: view:website:pos_journal_report.pos_journal_report_document
+msgid "- Troco"
```
Comment (legalsylvain) [inline pos_session_report/i18n/pt_BR.po]: could you regenerate this file ? this translation file contains unused terms.

File: pos_session_report/models/pos_session.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+# Hendrix Costa <hendrix.costa@kmee.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class PosSession(models.Model):
+    _inherit = 'pos.session'
+
+    user_id = fields.Many2one('res.users',
```
Comment (legalsylvain) [inline pos_session_report/models/pos_session.py]: why do you redefined this point_of_sale field ? 
https://github.com/odoo/odoo/blob/8.0/addons/point_of_sale/point_of_sale.py#L268

File: pos_session_report/report/pos_session_report.xml [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+                                    <span t-esc="statement.journal_id.name"/>
+                                </h4>
+                                <table class="table table-condensed">
+                                    <thead>
+                                        <tr>
+                                            <th>Name</th>
+                                            <th>Date</th>
+                                            <th>Pos Order</th>
+                                            <th>State</th>
+                                            <th>QTD Payments</th>
```
Comment (legalsylvain) [inline pos_session_report/report/pos_session_report.xml]: what is QTD ?

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi @pedrobaeza. No this module has nothing to do with pos_session_summary. 

* **pos_session_summary** add extra computed field on pos.session. (to know on the tree view, income evolution, number of order by session...)
* This module provides a new detailled qweb-pdf report to list all the activity of the session.

@Hendrixcosta : I do not understand the interest of your new qweb pdf, comparing the natif one. defined here :
https://github.com/odoo/odoo/commit/28fa8ebb8cb267ca3a30afba81aafb1eb9399894

You just seems to add a line in a native report, and for that, you redefine fully a new document. Inheritance should be used instead.

:-1: 
original pdf on the left // custom pdf on the right.

pos_session_report_comparison

**Comment by legalsylvain** [comment]
Inheritance mechanism not used. I'm closing this PR. Feel free to reopen if fixed.

**Pull request closed without merging**